### ISSUE TITLE
Colab中的cheakpoint未生效

### ISSUE THREAD

**Issue by DerilJust** [innnky/so-vits-svc#63]
2023-02-09 11:36:47,403	48k	INFO	====> Epoch: 210
2023-02-09 11:37:25,812	48k	INFO	====> Epoch: 211
2023-02-09 11:37:39,252	48k	INFO	Train Epoch: 212 [27%]
2023-02-09 11:37:39,253	48k	INFO	[2.314086675643921, 2.435959577560425, 9.16275691986084, 21.679513931274414, 0.7491365671157837, 15000, 9.73846430766616e-05]
2023-02-09 11:37:43,628	48k	INFO	Saving model and optimizer state at iteration 212 to ./logs/48k/G_15000.pth
2023-02-09 11:50:54,882	48k	INFO	{'train': {'log_interval': 200, 'eval_interval': 1000, 'seed': 1234, 'epochs': 10000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 12, 'fp16_run': False, 'lr_decay': 0.999875, 'segment_size': 17920, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0, 'use_sr': True, 'max_speclen': 384, 'port': '8001'}, 'data': {'training_files': 'filelists/train.txt', 'validation_files': 'filelists/val.txt', 'max_wav_value': 32768.0, 'sampling_rate': 48000, 'filter_length': 1280, 'hop_length': 320, 'win_length': 1280, 'n_mel_channels': 80, 'mel_fmin': 0.0, 'mel_fmax': None}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0.1, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 8, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'n_layers_q': 3, 'use_spectral_norm': False, 'gin_channels': 256, 'ssl_dim': 256, 'n_speakers': 2}, 'spk': {'zi': 0}, 'model_dir': './logs/48k'}
2023-02-09 11:51:02,928	48k	INFO	Loaded checkpoint './logs/48k/G_14000.pth' (iteration 198)
2023-02-09 11:51:12,853	48k	INFO	Train Epoch: 1 [0%]
2023-02-09 11:51:12,854	48k	INFO	[5.986483573913574, 5.219101428985596, 0.47303277254104614, 21.976987838745117, 0.9983282089233398, 0, 9.755522608029692e-05]
2023-02-09 11:51:18,038	48k	INFO	Saving model and optimizer state at iteration 1 to ./logs/48k/G_0.pth
2023-02-09 11:51:27,900	48k	INFO	Saving model and optimizer state at iteration 1 to ./logs/48k/D_0.pth
2023-02-09 11:52:09,366	48k	INFO	====> Epoch: 1
2023-02-09 11:52:48,748	48k	INFO	====> Epoch: 2

为什么在停止再开始之后是重新开始迭代而不是从上一个保存的版本开始

**Comment by DerilJust** [comment]
2023-02-09 11:37:43,628 48k INFO Saving model and optimizer state at iteration 212 to ./logs/48k/G_15000.pth
这里之后是意外中断的

**Comment by DerilJust** [comment]
再训练另一个模型的时候就好了
可能是Colab意外中断导致的
